src/types/cardano.ts

```diff
@@ -0,0 +1,169 @@
+import BigNumber from 'bignumber.js'
+
+// Wallet Candidates
+
+export type TransactionDirection = 'SENT' | 'RECEIVED' | 'SELF' | 'MULTI'
+export type TransactionStatus = 'SUCCESSFUL' | 'PENDING' | 'FAILED'
+export type TransactionAssurance = 'PENDING' | 'FAILED' | 'LOW' | 'MEDIUM' | 'HIGH'
+
+export type Block = {
+  height: number
+  epoch: number
+  slot: number
+  hash: string
+}
+
+export type TransactionInfo = {
```

Comment:
similar here
```ts
type TransactionInfo = {...} // cardano specific, combine with RawTransaction?
```

https://emurgo.atlassian.net/wiki/spaces/CARDANO/pages/618430503/Platform+Agnostic#Transaction
```ts
type Transaction = {
  ...all the same stuff except inputs, outputs...
}
```

Reply:
This will be the next step, for now just keeping the same types that we were using before.